src/main/java/com/todolist/app/springmvcboottodolist/controllers/SeeAllTasksController.java

```diff
@@ -0,0 +1,46 @@
+package com.todolist.app.springmvcboottodolist.controllers;
+
+
+import com.todolist.app.springmvcboottodolist.models.Item;
+import com.todolist.app.springmvcboottodolist.models.Status;
+import com.todolist.app.springmvcboottodolist.service.ToDoListStore;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.ui.Model;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+
+@Controller
+public class SeeAllTasksController {
+    @Autowired
+    ToDoListStore store;
+
+    @RequestMapping(value="/see-all-tasks",method = RequestMethod.GET)
+    public String getSeeAllTasks(Model model){
+        Map<String, Item> onhold = new LinkedHashMap<>();
```

Comment:
Could we extract such a logic of determining items with a proper status to the `Service`, e.g. `ToDoListService#getItemsByStatus(Status status)`

Don't forget to write a unit test for it